src/bundle/Core/Routing/DefaultRouter.php

```diff
@@ -16,30 +17,28 @@
 use Symfony\Bundle\FrameworkBundle\Routing\Router;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Exception\RouteNotFoundException;
-use Symfony\Component\Routing\Matcher\RequestMatcherInterface;
+use Symfony\Component\Routing\RequestContext;
 
 /**
  * Extension of Symfony default router implementing RequestMatcherInterface.
  */
-class DefaultRouter extends Router implements RequestMatcherInterface, SiteAccessAware
+class DefaultRouter extends Router implements SiteAccessAware
 {
-    /** @var \Ibexa\Core\MVC\Symfony\SiteAccess */
-    protected $siteAccess;
+    protected ?SiteAccess $siteAccess;
```

Comment:
Initialize it, so it doesn't need to be checked.

```suggestion
    protected ?SiteAccess $siteAccess = null;
```